src/ElectronicBankStatement.php

```diff
@@ -161,7 +142,11 @@ public function setTransactions(array $transactions): void
         $this->closevalue = $this->startvalue;
 
         foreach ($transactions as $transaction) {
-            $this->closevalue = $this->closevalue->add($transaction->getValue());
+            if ($transaction->getDebitCredit() == DebitCredit::CREDIT()) {
+                $this->closevalue = $this->closevalue->add($transaction->getValue());
+            } else {
+                $this->closevalue = $this->closevalue->subtract($transaction->getValue());
```

Comment:
Shouldn't you always `add()` the value, because the value is negative in case of a debit transaction?

Reply:
The value from a transaction is always a positive amount, the debit credit field is used to signal the sign.

Reply:
```
    public function getValue(): Money
    {
        /*
         * Always return the absolute value, Twinfield uses "debitcredit" fields instead of signs.
         */
        return $this->value->absolute();
    }
```